extensions/smallrye-health/deployment/src/test/java/io/quarkus/smallrye/health/test/ui/ErroneousConfigTest.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkus.smallrye.graphql.deployment.ui;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.deployment.configuration.ConfigurationError;
+import io.quarkus.test.QuarkusUnitTest;
+
+public class ErroneousConfigTest {
+
+    @RegisterExtension
+    static final QuarkusUnitTest config = new QuarkusUnitTest()
+            .setExpectedException(ConfigurationError.class)
+            .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+                    .addAsResource(new StringAsset("quarkus.smallrye-health.root-path-ui=/\n"), "application.properties"));
+
+    @Test
+    public void shouldNotStartApplicationIfUIPathIsASlash() {
+        Assertions.fail();
```

Comment:
probably forgotten implementation of the test :)

Reply:
The app should not startup, as it has invalid config. That is what is being test.